static/assets/readme/CircleCI.svg

```diff
@@ -1 +1,14 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="120" height="60" fill="#161616"><path d="M49.902 34.76h-1.524c-.102 0-.198.048-.26.13-.885 1.308-2.38 2.168-4.08 2.168-2.72 0-4.923-2.204-4.923-4.923s2.204-4.923 4.923-4.923a4.92 4.92 0 0 1 4.08 2.169c.062.08.158.13.26.13h1.524a.33.33 0 0 0 .293-.475 6.89 6.89 0 0 0-6.156-3.792c-3.806 0-6.893 3.086-6.893 6.893s3.086 6.893 6.893 6.893a6.89 6.89 0 0 0 6.155-3.792c.024-.046.036-.096.036-.147a.33.33 0 0 0-.329-.328zm4.595-11.816c0 1.75-2.626 1.75-2.626 0s2.626-1.75 2.626 0zM54.17 38.37v-12.8H52.2v12.8a.33.33 0 0 0 .328.328h1.313a.33.33 0 0 0 .328-.328zm8.54-13.12c-1.986.094-3.573 1.043-4.6 2.492v-1.844a.33.33 0 0 0-.328-.328H56.47a.33.33 0 0 0-.328.328V38.37c0 .18.146.328.328.328h1.313a.33.33 0 0 0 .328-.328v-6.236c0-2.6 2.03-4.743 4.595-4.912a.33.33 0 0 0 .329-.329v-1.316a.33.33 0 0 0-.323-.328zm13.08 9.5h-1.524c-.102 0-.198.048-.26.13-.885 1.308-2.38 2.168-4.08 2.168-2.72 0-4.923-2.204-4.923-4.923s2.204-4.923 4.923-4.923c1.698 0 3.194.86 4.08 2.17.062.08.158.13.26.13H75.8a.33.33 0 0 0 .329-.328.32.32 0 0 0-.036-.147 6.89 6.89 0 0 0-6.155-3.792c-3.807 0-6.893 3.086-6.893 6.893s3.086 6.893 6.893 6.893a6.89 6.89 0 0 0 6.155-3.792c.024-.046.036-.096.036-.147a.33.33 0 0 0-.329-.328zm3.94-13.457h-1.313a.33.33 0 0 0-.328.328v16.74a.33.33 0 0 0 .328.328h1.313a.33.33 0 0 0 .328-.328V21.63a.33.33 0 0 0-.328-.328zm9.2 3.94c-3.807 0-6.893 3.086-6.893 6.893s3.086 6.893 6.893 6.893a6.89 6.89 0 0 0 6.155-3.792c.113-.218-.047-.478-.292-.475H93.26c-.105 0-.203.05-.265.135-.885 1.304-2.38 2.162-4.075 2.162-2.496 0-4.558-1.858-4.88-4.267h11.434a.33.33 0 0 0 .327-.312l.01-.345c0-3.807-3.086-6.893-6.892-6.893zm-4.745 5.58c.575-2.08 2.48-3.6 4.745-3.6s4.17 1.53 4.745 3.6zm30.702-9.847c-2.626 0-2.626 3.94 0 3.94s2.626-3.94 0-3.94zm1.64 4.595v12.8a.33.33 0 0 1-.328.328h-2.626a.33.33 0 0 1-.328-.328v-12.8zM104.67 39.026c3.218 0 5.917-2.205 6.678-5.185a.3.3 0 0 0 .008-.067.33.33 0 0 0-.328-.328h-2.778a.33.33 0 0 0-.298.192c-.572 1.242-1.825 2.105-3.282 2.105a3.61 3.61 0 1 1 3.281-5.114.33.33 0 0 0 .298.191h2.778a.33.33 0 0 0 .328-.328.32.32 0 0 0-.008-.066c-.76-2.98-3.46-5.186-6.678-5.186-3.806 0-6.893 3.086-6.893 6.893s3.087 6.893 6.893 6.893z"/><g transform="matrix(1.395722 0 0 1.395722 38.606163 59.796655)"><circle cx="-14.275" cy="-21.349" r="2.73"/><path d="M-14.275-32.813c-5.342 0-9.83 3.654-11.103 8.598a.56.56 0 0 0-.019.136c0 .3.244.546.546.546h4.623a.54.54 0 0 0 .495-.32s.007-.013.01-.02c.953-2.056 3.034-3.483 5.45-3.483 3.317 0 6.006 2.688 6.006 6.005s-2.688 6.005-6.005 6.005c-2.415 0-4.496-1.427-5.45-3.483-.003-.007-.01-.02-.01-.02-.088-.194-.282-.32-.495-.32h-4.623c-.302 0-.546.244-.546.546a.54.54 0 0 0 .019.136c1.273 4.944 5.762 8.598 11.103 8.598C-7.942-9.89-2.8-15.022-2.8-21.353s-5.133-11.464-11.464-11.464z"/></g></svg>
\ No newline at end of file
+<svg
```

Comment:
These minified svgs aren't meant to be formatted by prettier (see both files)

Reply:
For reference which files need to be formatted by prettier you can look at lefthook.yml
![image](https://user-images.githubusercontent.com/20756439/197298830-de8c85c1-d303-4732-b9cf-cd99b6697427.png)

Reply:
They weren't. I formatted Vercel manually because I was messing around with its view box and forgot to change it.

CCI is formatted because I replaced the one introduced in #340 with the same one used in the sponsor section. The black levels were inconsistent between them. I just kept the original formatting.